### PR TITLE
[Topo] 3 port tgen topo for RDMA testing

### PR DESCRIPTION
**Pull request by developfast**
### Description of PR

Summary: Introducing a 3 port topo to run RDMA nightly tests on the switch. 
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [x] 201911
- [x] 202012
- [x] 202205

#### What is the motivation for this PR?
New topo needed for RDMA nightly testing on lab device. 
 
#### How did you do it?
Created a new topo file

### CODE REVIEW

File: ansible/vars/topo_tgen-t1-64-3.yml [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+      asn: 65200
+      peers:
+        64607:
+        - 1.0.0.2
+        - FC00::2
+    interfaces:
+      Loopback0:
+        ipv4: 100.1.0.3/32
+        ipv6: 2064:100::3/128
+      Ethernet1:
```
Comment (neethajohn) [inline ansible/vars/topo_tgen-t1-64-3.yml:110]: For the t1 topo, we would like to have at least 1 PortChannel interface. The other 1 can be regular interfaces

Comment (developfast) [reply ansible/vars/topo_tgen-t1-64-3.yml:110]: These are ixia devices - so the actual topo config does not matter. This is more for the sake of completeness.

Comment (neethajohn) [reply ansible/vars/topo_tgen-t1-64-3.yml:110]: We need to ensure that there are no issues when RDMA traffic enters/leaves portchannel interface/ pfc frames entering/exiting via member ports of a portchannel. Hence the need to simulate this

Comment (developfast) [reply ansible/vars/topo_tgen-t1-64-3.yml:110]: Done

**Pull request merged**